src/views/Login.vue

```diff
@@ -1,40 +1,25 @@
-<template>
-  <div class="container">
-    <img src="../assets/wolox-logo.png" class="logo">
-    <form @submit.prevent="onSubmit">
-      <div class="input-container">
-        <label for="email" class="label">Email</label>
-        <input
-          id="email"
-          v-model="email"
-          :class="{'input': true, 'error-input': ($v.email.$invalid && submitted) || !$v.email.email}"
-        >
-        <span class="error-label" v-if="!$v.email.email">El email no tiene un formato válido</span>
-        <span class="error-label" v-if="!$v.email.required && submitted">Campo requerido</span>
-      </div>
-      <div class="input-container">
-        <label for="password" class="label">Password</label>
-        <input
-          id="password"
-          type="password"
-          v-model="$v.password.$model"
-          :class="{'input': true, 'error-input': $v.password.$error && ($v.password.required || submitted )}"
-        >
-        <span class="error-label" v-if="!$v.password.required && submitted">Campo requerido</span>
-        <span
-          class="error-label"
-          v-if="$v.password.$dirty && $v.password.required && !$v.password.hasNumber"
-        >El password debe contener al menos un número</span>
-        <span
-          class="error-label"
-          v-if="$v.password.$dirty && $v.password.required && !$v.password.hasUppercase"
-        >El password debe contener al menos una mayúscula</span>
-      </div>
-      <button class="primary-button">Login</button>
-    </form>
-    <div class="horizontal-division"/>
-    <router-link to="/sign-up" class="secondary-button">Sign up</router-link>
-  </div>
+<template lang="pug">
+  .container
+    img.logo(src='../assets/wolox-logo.png', alt='wolox-books-logo')
```

Comment:
Instead of doing the attribute separation with commas, just leave a space :smile:

Reply:
Great! :wink::+1: